pyiron/vasp/potential.py

```diff
@@ -272,3 +277,187 @@ def find_potential_file(file_name=None, xc=None, path=None, pot_path_dict=None):
                     if file_name in file_lst:
                         return os.path.join(path, file_name)
     raise ValueError("Either the filename or the functional has to be defined.")
+
+
+def get_enmax_among_species(symbol_lst, return_list=False, xc="PBE"):
+    """
+    Given a list of species symbols, finds the largest applicable encut.
+
+    Args:
+        symbol_lst (list): The list of species symbols.
+        return_list (bool): Whether to return the list of all ENMAX values (in the same order as `species_lst` along with
+            the largest value). (Default is False.)
+        xc ("GGA"/"PBE"/"LDA"): The exchange correlation functional for which the POTCARs were generated. (Default is "PBE".)
```

Comment:
`xc (str): The exchange correlation functional for which the POTCARs were generated can have values of "GGA"/"PBE"/"LDA". (Default is "PBE".)`